packages/components/.eslintrc.js

```diff
@@ -1,3 +1,4 @@
+const path = require( 'path' );
```

Comment:
What's different about this package, why the special treatment?

Couldn't do something like this?

```js
module.exports = {
  ...generateForClient( __dirname )
  overrides: [
    // ...
  ]
};
```

Reply:
I think I just missed it.

Reply:
oh no, it's the bit about stories here. 

We could reuse bits of generateForClient, but it gets confusing (to me) what the actual output is in that case. Using the syntax you suggested, `overrides` is totally replaced and we'd still need to dupe stuff from the generator. I figured we might as well dupe all of it and make it explicit.

Reply:
This is the only package that includes stories, isn't it? That's why the additional overrides are here?

Reply:
or maybe we just make the stories config part of the standard...

why did we turn off the extraneous imports rule in the first place?

Reply:
Because stories are development-only and depend on (root) devDependencies, right?

Reply:
right, but we should be able to keep the extraneous imports rule, just with the correct config